types/enzyme/enzyme-tests.tsx

```diff
@@ -16,6 +16,7 @@ import { Component, ReactElement, ReactNode, HTMLAttributes, ComponentClass, Sta
 interface MyComponentProps {
     stringProp: string;
     numberProp: number;
+    functionProp(...args: string[]): boolean;
```

Comment:
Would like to this prop made optional so it's less invasive to the rest of the tests that don't care about invoke. If strictNullChecks is on, using ! is fine imo, or declaring a separate component/props for the invoke tests.

Reply:
I'll look into this, it felt bad updating so many tests that didn't rely on `functionProp`.